### DISCUSSION TITLE
1.8.0 - Issues with WeekIndex

### DISCUSSION THREAD

**Discussion by dhl220193** [MMRIZE/MMM-CalendarExt3#128] [Q&A]
I have been running MMM-CalendarExt3 perfectly for months. However, It appears that there's an issue with the WeekIndex value -1 in MMM-CalendarExt3 version 1.8.0. When set to -1, it shows two previous weeks instead of the expected one previous week, which should be WeekIndex: -2. Various values, such as 0 and -2, seem to work correctly, indicating that the problem specifically lies with WeekIndex: -1.

**Comment by eouia** [comment]
Thanks. There was some mistake. I fixed.
